packages/api-generator/src/locale/en/v-table.json

```
@@ -3,6 +3,6 @@
     "fixedHeader": "MISSING DESCRIPTION ([edit in github](https://github.com/vuetifyjs/vuetify/tree/master/packages/api-generator/src/locale/en/v-table.json))",
     "fixedFooter": "MISSING DESCRIPTION ([edit in github](https://github.com/vuetifyjs/vuetify/tree/master/packages/api-generator/src/locale/en/v-table.json))",
     "height": "MISSING DESCRIPTION ([edit in github](https://github.com/vuetifyjs/vuetify/tree/master/packages/api-generator/src/locale/en/v-table.json))",
-    "hover": "MISSING DESCRIPTION ([edit in github](https://github.com/vuetifyjs/vuetify/tree/master/packages/api-generator/src/locale/en/v-table.json))"
+    "hover": "If hover is true it will add a hover effect to a table's row when the mouse is over it."
```

```suggestion
    "hover": "Will add a hover effect to a table's row when the mouse is over it."
```

It sounded a bit repetitive with using hover twice.